src/DigitalTwins/Changelog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Onboard DigitalTwins to Private Link Common Cmdlets
```

Comment:
Let's update the product name to `Azure Digital Twins`

Reply:
I have removed the changelog. I am not entirely sure we should have any changelog since we have not done any changes for DigitalTwins module. I referred from previous PR - https://github.com/Azure/azure-powershell/pull/12251

Reply:
Hi @anakarparida , yes, please remove the changelog and re-target the pr to branch https://github.com/Azure/azure-powershell/tree/network-february so that your change will be released with network release.

Reply:
I have removed the changelog and retarget the pr to branch network-february